uint8 panPos
uint8 tiltPos
